### ISSUE TITLE
Windows 10 version requirement

### ISSUE THREAD

**Issue by jdevp** [crramirez/uwp_bridge_java#3]
hi crramirez,

I'm reading the link from the following. It says that 

"The Desktop Bridge was introduced in Windows 10, version 1607, and it can only be used in projects that target Windows 10 Anniversary Update (10.0; Build 14393) or a later release in Visual Studio."

I wonder if the restriction is just for VS or it also applies to this project.

https://docs.microsoft.com/en-us/windows/uwp/porting/desktop-to-uwp-manual-conversion

Also, at end of this project, we'll get an assigned App.appx ( SwingSet.appx in this case). But we'll still need to generate *.pri files,  Assets directory and package the application for the store, correct ?

The file launch4j.xml should be at the same structure level at app.exe and app.jar but in your example, it is under the root directory.

**Comment by crramirez** [comment]
Hello,

It applies to any Win32 application that you want to pack as an appx and
upload to the Store. Windows 10 Anniversary Update is the Windows version
which brings support to Win32 apps packed as appx.

To answer your question, yes this restriction applies to
desktop-to-uwp-manual-conversion
<https://docs.microsoft.com/en-us/windows/uwp/porting/desktop-to-uwp-manual-conversion>
as
well.

Yes, due to launch4j.xml should not be included in the final package. This
is only to generate the app.exe

Regards

**Comment by jdevp** [comment]
Hi,

Thanks for information. Is there a way to manually create appxupload package without VS ?  I read somewhere that MS Store only accepts appxupload package. Any other packages can only be used for testing purpose.

Thanks

**Comment by crramirez** [comment]
Yes of course follow the instructions I left in the README.md or
https://docs.microsoft.com/en-us/windows/uwp/porting/desktop-to-uwp-manual-conversion
 .

Check the scripts I have in the project

In fact I don't use VS

Regards

**Comment by jdevp** [comment]
Hi, Did you actually submit to Dev Center or just created App.appx for sideloading ? According to this link, I'll need to submit "desktop bridge form" for App.appx to be accepted.

http://answers.flyppdevportal.com/MVC/Post/Thread/2b2752c1-63a8-44b9-9caa-1c8d97e5d622?category=windowsstore

**Comment by crramirez** [comment]
I don't remember right now but I am sure I didn't use VS. Yes I had to fill
the form and pass a testing process very strict until my application was
accepted.

This is my application :

https://www.microsoft.com/store/apps/9MTC7F4TRZ0V

Good luck

**Comment by crramirez** [comment]
Hello,

As uploading my app again, I have a comment. `.appx` are accepted if your application is a win32 ported one.

Regards
